src/network/serverpackethandler.cpp

```diff
@@ -1443,6 +1455,9 @@ void Server::handleCommand_InventoryFields(NetworkPacket* pkt)
 		return;
 	}
 
+	// Player is active, so reset idle timer
```

Comment:
One more: these comments are quite superflours, it's obvious what `resetIdletime` does